src/components/Card.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { makeStyles } from '@material-ui/core/styles';
+import Paper from '@material-ui/core/Paper';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+
+const cardStyle = makeStyles(theme => ({
+  root: {
+    padding: theme.spacing(2, 1)
+  }
+}));
+
+const card = props => {
+  const classes = cardStyle();
+  const { title, body, type } = props.content;
+  const contentType = type === 'text' ? <Box>{body}</Box> : <Box>CHART</Box>;
```

Comment:
I'd probably call this variable `content`, as it really is just the content of the card :)

Reply:
Also: 👍